examples/Vanilla/OnLoad/main.js

```diff
@@ -11,13 +11,56 @@
 
   // Step 4: Register your hook
   // Caution: There may already be an onload registered - in which case use a decorator pattern.
-  window.onload = async () => {
+
+  async function timeout (ms) {
+    return new Promise(r => setTimeout(r, ms));
+  }
+
+  async function condition () {
+    return true;
```

Comment:
@robertdumitrescu DoD - Please restore the condition.

Reply:
DONE